services/motion/builtin/move_on_globe_test.go

```diff
@@ -0,0 +1,300 @@
+package builtin
```

Comment:
hooray for these new files!
edit: why not remove `builtin_test` in favor of `move_test.go`?

Reply:
Two reasons:
1) builtin_test tests a variety of different things in `builtin.go`, especially fusions of different features, it does not just test `Move`
2) were I to rename the file, the diff would be the entire file, making review of the changes I made to the tests in that file invisible.